hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/FullScanPhysicalRel.java

```diff
@@ -137,8 +141,44 @@ private static RelOptCost computeSelfCost(
 
     @Override
     public RelWriter explainTerms(RelWriter pw) {
+        final HazelcastTable hazelcastTable = OptUtils.extractHazelcastTable(this);
+        final var candidates = OptUtils.metadataQuery(this).extractPrunability(this);
```

Comment:
this will report full scan as pruned even though the member pruning is not used. However, together with https://github.com/hazelcast/hazelcast/commit/6f4aa7cb464fc2b697180e9df2828b6f58c3eab2 the output should be correct.

We might also report this in a misleading way for nested loops join - it might (?) happen that right hand table has predicates that make it eligible for pruning, but no pruning will be used because we use PredicateAPI for NLJ.

Reply:
for example what will the plan for such query: https://github.com/hazelcast/hazelcast/commit/8519c1c279df9f22c14868b4b497e35df5eabfc9?

Reply:
I was wrong - the output is incorrect. See `test_scanPruningWithoutMemberPruning`